Trie/Trie/Trie/Trie.swift

```diff
@@ -62,11 +60,32 @@ class Trie {
     fileprivate let root: Node
     fileprivate var wordCount: Int
 
-    /// Creats an empty trie.
-    init() {
+    /// Creates an empty trie.
+    override init() {
```

Comment:
I didn't know I needed to do that.  Doesn't the compiler complain if you need to call super.init()?